src/app/shared/services/auth/auth.service.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@angular/core';
+import { User } from '@interfaces/user';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private readonly KEY = 'user';
+
+  login(user: User): void {
```

Comment:
Although ok, you should use reactive approach:

https://blog.angular-university.io/how-to-build-angular2-apps-using-rxjs-observable-data-services-pitfalls-to-avoid/
with behaviour subject or sth like this:

https://ngrx.io/guide/component-store

i've discovered it recently

Reply:
About the reactive approach, I planned to make it reactive during HTTP module task.